### PR TITLE
Feature/medical screen

### PR DESCRIPTION
**Pull request by hb308**
# Description

Screen which takes in details about users medical details and their amputations. This is the second part of registering the users profile.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- Ran jest tests - none fail
- Tested using simular and iPhone 11

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by klbennett — APPROVED** [review]

File: mobile/src/components/atomic/AlertBox.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  hideAlert,
+}) => {
+  return (
+    <ViewContainer
+      backgroundColor="#fff"
+      justifyContent="center"
+      alignItems="center"
+    >
+      <AwesomeAlert
+        contentContainerStyle={{ justifyContent: 'stretch', width: 5000 }}
```
Comment (ndeasy) [inline mobile/src/components/atomic/AlertBox.js]: is 5000 right? seems like a lot

File: mobile/src/screens/MedicalScreen/components/AddAmputation.js [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+  const limbValues = [
+    {
+      title: 'Upper',
+      value: 'Upper',
+    },
+    {
+      title: 'Lower',
+      value: 'Lower',
+    },
+  ];
```
Comment (ndeasy) [inline mobile/src/screens/MedicalScreen/components/AddAmputation.js:120]: I would move these to your Amputation helper in Utils

File: mobile/src/components/atomic/AlertBox.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  showConfirmButton,
+}) => {
+  return (
+    <ViewContainer
+      backgroundColor="#fff"
+      justifyContent="center"
+      alignItems="center"
+    >
+      <AwesomeAlert
+        contentContainerStyle={{ width: 5000 }}
```
Comment (ndeasy) [inline mobile/src/components/atomic/AlertBox.js:25]: Should this be 5000 ?

Comment (hb308) [reply mobile/src/components/atomic/AlertBox.js:25]: let me take a look at that

File: mobile/src/screens/MedicalScreen/components/AddAmputation.js (hunk 2 above)
Comment (hb308) [reply mobile/src/screens/MedicalScreen/components/AddAmputation.js:120]: coolio

File: mobile/src/components/atomic/AlertBox.js (hunk 3 above)
Comment (hb308) [reply mobile/src/components/atomic/AlertBox.js:25]: Changed it to 'auto' so it adjusts based on the content

**Pull request merged**